app/templates/components/wikia-ui-components/wiki-page-header.hbs

```diff
@@ -8,7 +8,7 @@
 				{{#link-to 'wiki-page' mainPageTitle title=siteName class='wiki-page-header__site-link'}}
 					<span class="wiki-page-header__site-name">{{siteName}}</span>
 					{{i18n 'app.header-in-wiki-name-label-suffix'}}
-					{{svg 'arrow-right-long'}}
+					{{svg 'wds-icons-arrow' class='wds-icon wds-icon-tiny'}}
```

Comment:
if this should be tiny icon then use wds-icons-arrow-tiny, otherwise why we apply wds-icon-tiny class?